build/karma.sauce.config.js

```diff
@@ -90,6 +90,9 @@ module.exports = function (config) {
     sauceLabs: {
       testName: 'Vue.js unit tests',
       recordScreenshots: false,
+      connectOptions: {
+        'no-ssl-bump-domains': 'all', // Ignore SSL error on Android emulator
```

Comment:
It could be made more specific:
`'no-ssl-bump-domains': '*google.com'`
...in my testing, it's only the Android 4 emulator.